cmd/api/src/test/integration/harnesses.go

```diff
@@ -1251,7 +1252,77 @@ func (s *ADCSESC1Harness) Setup(graphTestContext *GraphTestContext) {
+	emptyAppPolicies := make([]string, 0)
+	sid = RandomDomainSID()
+	s.Domain2 = gt.NewActiveDirectoryDomain("domain2", sid, false, true)
+	s.AuthStore2 = gt.NewActiveDirectoryNTAuthStore("authstore2", sid)
+	s.RootCA2 = gt.NewActiveDirectoryRootCA("rca2", sid)
+	s.EnterpriseCA2 = gt.NewActiveDirectoryEnterpriseCA("eca2", sid)
+	s.CertTemplate21 = gt.NewActiveDirectoryCertTemplate("certtemplate2-1", sid, false, false, false, false, 1, 0, certRequestAgentEKU, emptyAppPolicies)
+	s.CertTemplate22 = gt.NewActiveDirectoryCertTemplate("certtemplate2-2", sid, false, false, false, false, 1, 0, []string{adAnalysis.EkuCertRequestAgent, adAnalysis.EkuAnyPurpose}, emptyAppPolicies)
+	s.CertTemplate23 = gt.NewActiveDirectoryCertTemplate("certtemplate2-3", sid, false, false, false, false, 2, 1, certRequestAgentEKU, []string{adAnalysis.EkuCertRequestAgent})
+	s.CertTemplate24 = gt.NewActiveDirectoryCertTemplate("certtemplate2-4", sid, false, false, false, false, 2, 1, []string{}, emptyAppPolicies)
+	s.CertTemplate25 = gt.NewActiveDirectoryCertTemplate("certtemplate2-5", sid, false, false, false, true, 1, 1, []string{}, emptyAppPolicies)
+
+	gt.NewRelationship(s.AuthStore2, s.Domain2, ad.NTAuthStoreFor)
+	gt.NewRelationship(s.RootCA2, s.Domain2, ad.RootCAFor)
+	gt.NewRelationship(s.EnterpriseCA2, s.AuthStore2, ad.TrustedForNTAuth)
+	gt.NewRelationship(s.EnterpriseCA2, s.RootCA2, ad.EnterpriseCAFor)
+	gt.NewRelationship(s.CertTemplate21, s.EnterpriseCA2, ad.PublishedTo)
+	gt.NewRelationship(s.CertTemplate22, s.EnterpriseCA2, ad.PublishedTo)
+	gt.NewRelationship(s.CertTemplate23, s.EnterpriseCA2, ad.PublishedTo)
+	gt.NewRelationship(s.CertTemplate24, s.EnterpriseCA2, ad.PublishedTo)
+	gt.NewRelationship(s.CertTemplate25, s.EnterpriseCA2, ad.PublishedTo)
+}
+
+type EnrollOnBehalfOfHarnessOne struct {
+	Domain1        *graph.Node
+	AuthStore1     *graph.Node
+	RootCA1        *graph.Node
+	EnterpriseCA1  *graph.Node
+	CertTemplate11 *graph.Node
+	CertTemplate12 *graph.Node
+	CertTemplate13 *graph.Node
+}
+
+func (s *EnrollOnBehalfOfHarnessOne) Setup(gt *GraphTestContext) {
+	sid := RandomDomainSID()
+	anyPurposeEkus := make([]string, 0)
+	anyPurposeEkus = append(anyPurposeEkus, adAnalysis.EkuAnyPurpose)
+	emptyAppPolicies := make([]string, 0)
+	s.Domain1 = gt.NewActiveDirectoryDomain("domain1", sid, false, true)
+	s.AuthStore1 = gt.NewActiveDirectoryNTAuthStore("authstore1", sid)
+	s.RootCA1 = gt.NewActiveDirectoryRootCA("rca1", sid)
+	s.EnterpriseCA1 = gt.NewActiveDirectoryEnterpriseCA("eca1", sid)
+	s.CertTemplate11 = gt.NewActiveDirectoryCertTemplate("certtemplate1-1", sid, false, false, false, false, 2, 0, anyPurposeEkus, emptyAppPolicies)
+	s.CertTemplate12 = gt.NewActiveDirectoryCertTemplate("certtemplate1-2", sid, false, false, false, false, 1, 0, anyPurposeEkus, emptyAppPolicies)
+	s.CertTemplate13 = gt.NewActiveDirectoryCertTemplate("certtemplate1-3", sid, false, false, false, false, 2, 0, anyPurposeEkus, emptyAppPolicies)
```

Comment:
The schemaversion for certtemplate1-1 does not match the property declared in the json/svg representation.